lib/ansible/modules/database/mysql/mysql_db.py

```diff
@@ -294,6 +330,7 @@ def main():
     db = module.params["name"]
     if not db:
         module.fail_json(msg="Please provide at least one database name")
```

Comment:
Since a related thing came up in the Ansible project meeting: modules should not fail if nothing is passed. (At least where it makes sense, and here it does: if no DBs are specified, there's simply no change to do.)